paddle/fluid/framework/ir/conv_bn_fuse_pass.cc

```diff
@@ -136,18 +136,21 @@ void ConvBNFusePass::ApplyImpl(ir::Graph* graph) const {
       return;
     }
 
+    // Get batch norm bias
+    auto* bn_bias_tensor =
+        scope->FindVar(bn_bias->Name())->GetMutable<LoDTensor>();
+
     // Create eltwise_y (conv bias) variable
     VarDesc eltwise_y_in_desc(
         patterns::PDNodeName(name_scope_, "eltwise_y_in"));
+    eltwise_y_in_desc.SetShape(framework::vectorize(bn_bias_tensor->dims()));
+    eltwise_y_in_desc.SetDataType(bn_bias_tensor->type());
+    eltwise_y_in_desc.SetLoDLevel(0);
```

Comment:
Why do you need to add line146-148?

Reply:
commented here https://github.com/PaddlePaddle/Paddle/pull/17143#issuecomment-487828409

Reply:
`eltwise_y_in_desc.SetLoDLevel(0);`->`eltwise_y_in_desc.SetLoDLevel(bn_bias_tensor->lod_level())`;?

Reply:
`bn_bias_tensor ` do not have `lod_level()`

but I can use bn_bias_var

Reply:
done